outputs/jira.go

```diff
@@ -439,3 +460,11 @@ func isServerJira(rawUrl string) bool {
 
 	return false
 }
+
+func cpyUnknowns(source map[string]string) map[string]string {
```

Comment:
It is `copy`, if I understood correctly

Reply:
it was attempt to make method name more short